### ISSUE TITLE
Trending repos are not showing 

### ISSUE THREAD

**Issue by philoez98** [ThirtyDegreesRay/OpenHub#269]
Is it maybe a html parsing issue? Heard that github changed some html tags, so it might have messed it up.

**Comment by scarab5q** [comment]
Yeah I have the same issue, might be to do with HTML or something to do with the GitHub API as the same thing happens on fasthub

**Comment by k0shk0sh** [comment]
that's what happen when you copy FastHub code @ThirtyDegreesRay  :D.

**Comment by philoez98** [comment]
@k0shk0sh LOL. Btw when we will have a fix on FastHub?

**Comment by StrongerL** [comment]
I have the same problem

**Comment by smilebinbin** [comment]
I have the same problem！when to fix the bug?

**Comment by christophegrall** [comment]
work in progress

**Comment by christophegrall** [comment]
fixed : https://github.com/ThirtyDegreesRay/OpenHub/pull/274

**Comment by christophegrall** [comment]
https://github.com/christophegrall/OpenHub/releases/tag/v3.0.1-alpha

**Comment by phammanh029** [comment]
i saw that something maybe miss match. So i try to make it work by changing some code. Please look at:
[](https://github.com/phammanh029/OpenHub/commit/21a2cb6a7c75b5f43fc334e78b199ce7cc5a72a9)

if this is acceptable. Feel free to merge it to the repo.

Many thanks for great team.
